### ISSUE TITLE
Typo in Documentation

### ISSUE THREAD

**Issue by fantasydr** [newren/git-filter-repo#148]
I think the command parameter should be
```
--use-base-name
```

But in https://github.com/newren/git-filter-repo/blob/main/Documentation/converting-from-bfg-repo-cleaner.md
```
git filter-repo --use-base-names --path id_dsa --path id_rsa --invert-paths
```
This will cause ```unrecognized arguments: --use-base-names``` error message.

**Comment by fantasydr** [comment]
BTW, I have to modfy the source code line 2909:
from ```for root, dirs, files in os.walk(reflog_dir):```
to ```for root, dirs, files in os.walk(str(reflog_dir, 'utf-8')):```
Then it could run on Win10 platform, otherwise there was error message
```
os.scandir() doesn't support bytes path on Windows, use Unicode instead
```
Since I'm not very familiar with Python3, I'll just leave a message here other than a pull request.

**Comment by newren** [comment]
Thanks for reporting this.  Someone sent in a patch for this in #151 and hopefully they'll fix up the minor things needed in the commit message.

I guess I shouldn't be surprised that windows (or at least python on windows) fails when given bytestrings for pathnames.  What a pain.  I'm worried there'll be a lot of other errors that need to be cleaned out too, and I'm kind of surprised we haven't seen more errors already though -- why aren't there lots of reports of this?  Anyway, I'll try to get it fixed up, but I won't have a way to test it.

Sorry for taking so long to get back to this...

**Comment by newren** [comment]
Fixed by Tom Matthews in commit 96959d1174 (converting-from-bfg-repo-cleaner.md: fix typo, 2020-09-01).
